libcontainer/configs/validate/validator.go

```diff
@@ -406,5 +406,13 @@ func ioPriority(config *configs.Config) error {
 	if priority < 0 || priority > 7 {
 		return fmt.Errorf("invalid ioPriority.Priority: %d", priority)
 	}
+
+	switch config.IOPriority.Class {
+	case specs.IOPRIO_CLASS_RT, specs.IOPRIO_CLASS_BE, specs.IOPRIO_CLASS_IDLE:
+		// do nothing
```

Comment:
I don't like this comment, but this is my personal preference. Maybe should change to `// should return nil` or move to an new function:
```go
func checkIOPriorityClass(specs.IOPriorityClass) bool {
    switch config.IOPriority.Class {
	case specs.IOPRIO_CLASS_RT, specs.IOPRIO_CLASS_BE, specs.IOPRIO_CLASS_IDLE:
            return true
    }
    return false
}
```

Reply:
Changed the comment to ` // Valid class, do nothing.`, and the whole switch to
```go
          switch class := config.IOPriority.Class; class {                    
          case specs.IOPRIO_CLASS_RT, specs.IOPRIO_CLASS_BE, specs.IOPRIO_CLASS_IDLE:
                  // Valid class, do nothing.
          default:
                  return fmt.Errorf("invalid ioPriority.Class: %q", class)                                             
          }
```